docker_101/docker_101.md

```diff
@@ -0,0 +1,427 @@
+<!--
+
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: wrapper
+docs_version: 1.0.0
+language: en
+narrator: UK English Female
+mode: Textbook
+
+title: Getting Started with Docker for Research
+
+comment: This tutorial combines a hands-on interactive Docker tutorial published by Docker Inc with an academic article outlining best practices for using Docker for research. 
+
+long_description: If you've been curious about how to use Docker for your research, this module is a great place to start. The Docker 101 tutorial is a popular, hands-on approach to learning Docker that will get you using containers right away, so you can learn by doing. To help you bridge the gap between basic Docker use and best practices for using Docker in research, we also link to an article outlining 10 rules to help you create great containers for research, and lists of ready-to-use Docker images for a variety of analysis workflows. This module includes running and editing comamnds in the terminal, so you'll need some basic familiarity with bash, but it is otherwise appropriate for beginniners. No prior experience with Docker or containers is assumed. 
```

Comment:
```suggestion
long_description: If you've been curious about how to use Docker for your research, this module is a great place to start. The Docker 101 tutorial is a popular, hands-on approach to learning Docker that will get you using containers right away, so you can learn by doing. To help you bridge the gap between basic Docker use and best practices for using Docker in research, we also link to an article outlining 10 rules to help you create great containers for research, and lists of ready-to-use Docker images for a variety of analysis workflows. This module includes running and editing commands in the terminal, so you'll need some basic familiarity with bash, but it is otherwise appropriate for beginners. No prior experience with Docker or containers is assumed. 
```